build/index.html

```diff
@@ -1,15 +1,218 @@
+              </a>
+            </li>
+            <li class="page-nav__item">
+              <a class="page-nav__link" href="#">
+                <span class="page-nav__text">Контакты</span>
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+      <picture>
+        <source type="image/webp" media="(min-width: 1366px)" srcset="img/header-bg-desktop@1x.webp, img/header-bg-desktop@2x.webp 2x">
+        <source type="image/webp" media="(min-width: 768px)" srcset="img/header-bg-tablet@1x.webp, img/header-bg-tablet@2x.webp 2x">
+        <source type="image/webp" srcset="img/header-bg-mobile@1x.webp, img/header-bg-mobile@2x.webp 2x">
+
+        <source media="(min-width: 1366px)" srcset="img/header-bg-desktop@1x.jpg, img/header-bg-desktop@2x.jpg 2x">
+        <source media="(min-width: 768px)" srcset="img/header-bg-tablet@1x.jpg, img/header-bg-tablet@2x.jpg 2x">
+        <img src="img/header-bg-mobile@1x.jpg" srcset="img/header-bg-mobile@2x.jpg" alt="Антарктический пейзаж">
+      </picture>
+    </header>
+
+    <main class="page-main">
+      <h1 class="visually-hidden">Круизы в Антарктикую. Волшебный мир айсбергов и пингвинов</h1>
+      <section class="main-header">
+        <h2 class="visually-hidden">Приветствующая секция</h2>
+        <div class="main-header__wrapper">
+          <dl class="main-header__list">
+            <dt class="main-header__title">Круизы в Антарктику</dt>
+            <dd class="main-header__description">Волшебный мир айсбергов и пингвинов</dd>
+          </dl>
+        </div>
+      </section>
+
+      <section class="main-description">
+        <h2 class="visually-hidden">Секция с описанием путешествия</h2>
+        <div class="main-description__wrapper">
+          <dl class="main-description__list">
+            <dt class="main-description__title">Путешествие на край Земли</dt>
+            <dd class="main-description__text">Это волшебное путешествие поможет вам познакомиться поближе с удивительным миром Антарктики. Наш круизный лайнер отвезет вас на край Земли, где вы сможете увидеть: ледяные айсберги, несколько видов пингвинов, китов, морских котиков и многое другое.</dd>
+          </dl>
+          <picture>
+            <source type="image/webp" media="(min-width: 1366px)" srcset="img/penguins-desktop@1x.webp, img/penguins-desktop@2x.webp 2x">
+            <source type="image/webp" media="(min-width: 768px)" srcset="img/penguins-tablet@1x.webp, img/penguins-tablet@2x.webp 2x">
+            <source type="image/webp" srcset="img/penguins-mobile@1x.webp, img/penguins-mobile@2x.webp 2x">
+    
+            <source media="(min-width: 1366px)" srcset="img/penguins-desktop@1x.jpg, img/penguins-desktop@2x.jpg 2x">
+            <source media="(min-width: 768px)" srcset="img/penguins-tablet@1x.jpg, img/penguins-tablet@2x.jpg 2x">
+            <img src="img/penguins-mobile@1x.jpg" srcset="img/penguins-mobile@2x.jpg" alt="Пингвинчики">
+          </picture>
```

Comment:
лучше поменять порядок на desktop first
```suggestion
         <picture>
          <source type="image/webp" media="(max-width: 500px)"  srcset="img/penguins-mobile@1x.webp, img/penguins-mobile@2x.webp 2x">
          <source type="image/jpg" media="(max-width: 500px)"  srcset="img/penguins-mobile@1x.jpg, img/penguins-mobile@2x.jpg 2x">
          <source type="image/webp" media="(max-width: 1200px)" srcset="img/penguins-tablet@1x.webp, img/penguins-tablet@2x.webp 2x">
          <source type="image/jpg" media="(max-width: 1200px)" srcset="img/penguins-tablet@1x.jpg, img/penguins-tablet@2x.jpg 2x">
          <source type="image/webp" srcset="img/penguins-desktop@1x.webp, img/penguins-desktop@2x.webp 2x">
          <img src="img/penguins-desktop@1x.jpg" srcset="img/penguins-desktop@2x.jpg 2x" alt="Пингвинчики">
        </picture>
```